lib/bpk-component-banner-alert/src/AnimateAndFade.js

```diff
@@ -101,18 +101,16 @@ class AnimateAndFade extends Component<Props> {
     this.measure(captureHeightAndAnimate);
   }
 
-  UNSAFE_componentWillReceiveProps(nextProps: Props) {
+  componentDidUpdate(prevProps: Props) {
+    this.measure(this.animate);
+
     this.shouldRenderHeight = true;
 
-    if (nextProps.show === this.props.show && this.props.show) {
+    if (this.props.show === prevProps.show && prevProps.show) {
```

Comment:
Isn't this doing the same as before?

Reply:
Yeah.. isn't that the point? I'm just trying to preserve the original behaviour as closely as possible. Maybe I've misunderstood your question 😅

Reply:
Also why isn't it just `this.props.show && prevProps.show`?

Reply:
I honestly don't know, I literally just preserved what was already there and just changed it to use `prevProps`. 

Wasn't it you who wrote this component initially? :P

Reply:
> Wasn't it you who wrote this component initially?
I can neither confirm nor deny that.

Here's the boolean logic:
![IMG_5713](https://user-images.githubusercontent.com/30267516/89648872-d0af8500-d8b7-11ea-9736-e0003da0b63f.jpg)